### PR TITLE
Skios 180 subscriptions loading time to long if a feed with many articles is subscribed

### PR DESCRIPTION
**Pull request by LukasLJL**
# Skiosa - PR
- [ ] added test(s), or explain why one is not needed/possible...
- [x] added comments to all the functions
- [ ] added documentation to latex project
- [x] code can't be split up into smaller logical units
- [x] code is formated and linted

# Notes
- Please merge after https://github.com/skiosa/frontend/pull/31 because this branch comes from #31
- Added some Pagination for Subscription and Similar Articles
- Also Added the Option to use the ``Desc``mode which fetches the data from our api in desc order, by the published-Date of an article.

### CODE REVIEW

File: src/app/pages/bookmark/bookmark.component.ts [hunk 1]
```diff
@@ -28,6 +33,7 @@ export class BookmarkComponent implements OnInit {
 			})
 			.valueChanges.subscribe(({ data }) => {
 				this.bookmarks = data.bookmarks;
+				console.log(this.bookmarks);
```
Comment (simonmorgenstern) [inline src/app/pages/bookmark/bookmark.component.ts]: brauchen wir die logs für tests? :) falls ja, kann man den log evtl noch ein wenig infomativer machen? timestamp + file name? :)

Comment (LukasLJL) [reply src/app/pages/bookmark/bookmark.component.ts]: vergessen zu entfernen :D

Comment (LukasLJL) [reply src/app/pages/bookmark/bookmark.component.ts]: nvm. habs es vor 5min removed

**Review by simonmorgenstern — APPROVED** [review]
thanks :)

**Review by simonmorgenstern — APPROVED** [review]

File: src/app/core/queries/subscription.ts [hunk 2]
```diff
@@ -20,16 +21,16 @@ export type SUBSCRIPTION_QUERY_RESPONSE = {
 	}[];
 };
 
-export const SUBSCRIPTION_QUERY = gql<SUBSCRIPTION_QUERY_RESPONSE, {}>`
-	query subscriptions {
+export const SUBSCRIPTION_QUERY = gql<SUBSCRIPTION_QUERY_RESPONSE, { PaginationArg: PaginationArg; Desc: boolean }>`
```
[the remark this answers is not in the document]
Comment (LukasLJL) [reply src/app/core/queries/subscription.ts]: done :D

**Pull request merged**